src/components/Card.spec.js

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from '@testing-library/react'
+import Card from './Card'
+
+describe('Card', () => {
+  it('renders name of card', () => {
+    render(<Card title="Völkerschlachtdenkmal" />)
+    expect(screen.getByText('Völkerschlachtdenkmal')).toBeInTheDocument()
+  })
```

Comment:
I would test all props and with the right role.
```suggestion
  it('renders name of card', () => {
    render(<Card title="Völkerschlachtdenkmal" text="Lorem Ipsum"  image="path/to/image" ratingImage="path/to/Image" />)
    expect(screen.getByRole('heading', {name: 'Völkerschlachtdenkmal'})).toBeInTheDocument()
    ....
  })
```